samples/interactive-tutorials/user_environment_setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright 2022 Google Inc. All Rights Reserved.
```

Comment:
Can you check both these scripts on https://www.shellcheck.net/ and accommodate the suggestions there?

Reply:
done